packages/webpack/src/plugin.js

```diff
@@ -339,6 +353,37 @@ class LavaMoatPlugin {
                   return
                 }
 
+                // Fixes the issue with assets being emitted to dist without a loader
+                // TODO: refactor to move random hardening of the build somewhere it's easier to track.
+                if (
+                  isAmbientAsset(module) &&
+                  module.resource.includes('node_modules') // FIXME: would be better to use canonicalName lookup and match with root
+                ) {
+                  // add a warning about removing the asset
+                  mainCompilationWarnings.push(
+                    new WebpackError(
+                      `LavaMoatPlugin: the following resource was being silently emitted to the dist and LavaMoat has prevented it: '${module.resource}'. If you want to add this resource, explicitly define a file-loader for it in your webpack configuration.`
+                    )
+                  )
+
+                  // chunkGraph.disconnectChunkAndModule(chunk, module) // can't use this, the require statement remains and errors out
```

Comment:
```suggestion
                  // We can't use `chunkGraph.disconnectChunkAndModule` here
                  // because the require statement remains and errors out
```